### PR TITLE
feat:fill in the requestHeader with exclusionOperationMap

### PR DESCRIPTION
**Pull request by coryhh**

### CODE REVIEW

**Comment by mokdeve** [comment]
Non-standard headers can be used, with the convention of prefixing the header name with `X-`, refer [here](https://en.wikipedia.org/wiki/List_of_HTTP_header_fields#cite_note-0) for more info.

Use `X-AREX-Exclusion-Operations` instead of `exclusionOperationMap`

File: src/main/java/com/arextest/replay/schedule/sender/impl/DefaultHttpServletReplaySender.java [hunk 1]
```diff
@@ -123,6 +123,9 @@ public boolean send(ReplayActionCaseItem caseItem) {
         before(caseItem.getRecordId());
         headers.remove(CommonConstant.AREX_REPLAY_WARM_UP);
         headers.put(CommonConstant.AREX_RECORD_ID, caseItem.getRecordId());
+        if (caseItem.getParent() != null && StringUtils.isNotEmpty(caseItem.getParent().getExclusionOperationConfig())) {
```
Comment (mokdeve) [inline src/main/java/com/arextest/replay/schedule/sender/impl/DefaultHttpServletReplaySender.java:126]: use a local var for `caseItem.getParent().getExclusionOperationConfig())`

Comment (coryhh) [reply src/main/java/com/arextest/replay/schedule/sender/impl/DefaultHttpServletReplaySender.java]: done

File: src/main/java/com/arextest/replay/schedule/service/PlanConsumeService.java [hunk 2]
```diff
@@ -161,14 +161,14 @@ private int streamingCaseItemSave(ReplayActionItem replayActionItem) {
...
-            size = doFixedCaseSave(caseItemList);
+            size = doFixedCaseSave(caseItemList, replayActionItem);
         } else {
             size = doPagingLoadCaseSave(replayActionItem);
         }
         return size;
     }
 
-    private int doFixedCaseSave(List<ReplayActionCaseItem> caseItemList) {
+    private int doFixedCaseSave(List<ReplayActionCaseItem> caseItemList, ReplayActionItem replayActionItem) {
```
Comment (binyu1005) [inline src/main/java/com/arextest/replay/schedule/service/PlanConsumeService.java:171]: 移除未使用的变量 replayActionItem

Comment (coryhh) [reply src/main/java/com/arextest/replay/schedule/service/PlanConsumeService.java]: done

**Review by binyu1005 — APPROVED** [review]

**Pull request merged**